magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -207,12 +184,34 @@ def mc_dl0_to_dl1(input_file, output_dir, config):
 
     zenith = 90 - sim_config["max_alt"].to_value("deg")
     azimuth = Angle(sim_config["max_az"]).wrap_at("360 deg").degree
-
+    logger.info(np.asarray(list(assigned_tel_ids.values())))
+    LSTs_IDs = np.asarray(list(assigned_tel_ids.values())[0:4])
+    LSTs_in_use = np.where(LSTs_IDs > 0)[0] + 1   #Here we select which LSTs are/is in use
```

Comment:
doesn't this also depend on 1,2, 3, ... numbering?

Reply:
See the previous reply about the check_input_list function.